src/env.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module "typescript" {
+  // https://github.com/microsoft/TypeScript/blob/81c951894e93bdc37c6916f18adcd80de76679bc/src/compiler/scanner.ts#L985
+  export function isIdentifierText(
```

Comment:
This change will affect all ts programs who somehow end up loading prisma-kysely. Even though none should be doing it, I've had some problems with it in the past. I think `const isIdentifierText = ts.isIdentifierText` is better in this case.

Reply:
I'm fairly confident that's not the case - it will only do that if it's included in the actual build. This is because .d.ts files are not automatically loaded like that in node_modules

With that said, it's worth me trying and packing a build just to make sure.

Reply:
Anyone who wants to extend prisma-kysely will load this declaration. Again, super small edge case here. I remember having problems with extending `"typescript"` module a while ago with either prisma-json-types-generator or ts-json-schema-generator. That's why I'm hesitant in this :)

Reply:
The reference has to exist somewhere in the module - this is why Bun published `bun-types` for a while and we had users include it in their `.types` array in tsconfig. 

However since this should only exist in our source code we can move it to the project root for example. That way it wouldn't be included in the published tarball and we can be certain it's safe.

Reply:
> The reference has to exist somewhere in the module

The beauty of `const isIdentifierText = ts.isIdentifierText` exists by not overriding any type definitions.

I'm ok with any implementation as long as its not included in the transpiled definition and/or final tarball.